tests/ManualModeTest.php

```diff
@@ -0,0 +1,42 @@
+<?php
+
+declare(strict_types=1);
+
+use Illuminate\Support\Facades\Event;
+use Stancl\JobPipeline\JobPipeline;
+use Stancl\Tenancy\Events\TenancyEnded;
+use Stancl\Tenancy\Events\TenancyInitialized;
+use Stancl\Tenancy\Events\TenantCreated;
+use Stancl\Tenancy\Jobs\CreateDatabase;
+use Stancl\Tenancy\Listeners\CreateTenantConnection;
+use Stancl\Tenancy\Listeners\UseCentralConnection;
+use Stancl\Tenancy\Listeners\UseTenantConnection;
+use \Stancl\Tenancy\Tests\Etc\Tenant;
+
+test('manual tenancy initialization works', function () {
+    Event::listen(TenantCreated::class, JobPipeline::make([CreateDatabase::class])->send(function (TenantCreated $event) {
+        return $event->tenant;
+    })->toListener());
```

Comment:
> Do we need this? And the createUsersTable() call below

Not really. But it makes sure the database connection is created and can be used, actually.

Reply:
> Do we need this? And the createUsersTable() call below

Without this, the below assertion got failed. Because tenant connection is not actually used so it's not in the connections class yet. 
```php
expect(array_keys(app('db')->getConnections()))->toBe(['central', 'tenant']);
```
Removing this doesn't effect much but it's extra check which is good to keep.

Reply:
I see. Then just add a comment explaining that it's to trigger the creation of the `'tenant'` connection.